docs/framework/network-programming/tls.md

```diff
@@ -15,34 +15,57 @@ helpviewer_keywords:
+| 4.6.2                         | TLS 1.2    | TLS 1.3    |
+| 4.7                           | TLS 1.2    | TLS 1.3    |
+| 4.7.1                         | TLS 1.2    | TLS 1.3    |
+| 4.7.2                         | TLS 1.2    | TLS 1.3    |
+| 4.8                           | TLS 1.2    | TLS 1.3    |
+| 4.8.1                         | TLS 1.2    | TLS 1.3    |
+
+For more information see [TLS protocol version support in Schannel](https://learn.microsoft.com/en-us/windows/win32/secauthn/protocols-in-tls-ssl--schannel-ssp-#tls-protocol-version-support).
 
-- For TLS 1.2, target .NET Framework 4.7 or later versions on your apps, and target .NET Framework 4.7.1 or later versions on your WCF apps.
-- For TLS 1.3, target .NET Framework 4.8 or later.
+## Recommendations
+
+- For TLS 1.3, target .NET Framework 4.6.2 or later.
 - Do not specify the TLS version. Configure your code to let the OS decide on the TLS version.
-- Perform a thorough code audit to verify you're not specifying a TLS or SSL version.
+- Perform a thorough code audit to verify you're not specifying a TLS or SSL version explicitly.
+- Do not use `SslProtocols.Default`.
 
 When your app lets the OS choose the TLS version:
 
-- It automatically takes advantage of new protocols added in the future, such as TLS 1.3.
+- It automatically takes advantage of new protocols added in the future.
 - The OS blocks protocols that are discovered not to be secure.
 
 The section [Audit your code and make code changes](#audit-your-code-and-make-code-changes) covers auditing and updating your code.
 
-This article explains how to enable the strongest security available for the version of .NET Framework that your app targets and runs on. When an app explicitly sets a security protocol and version, it opts out of any other alternative, and opts out of .NET Framework and OS default behavior. If you want your app to be able to negotiate a TLS 1.2 connection, explicitly setting to a lower TLS version prevents a TLS 1.2 connection.
+This article explains how to enable the strongest security available for the version of .NET Framework that your app targets and runs on. When an app explicitly sets a security protocol and version, it opts out of any other alternative, and opts out of .NET Framework and OS default behavior. If you want your app to be able to negotiate a TLS 1.3 connection, explicitly setting to a lower TLS version prevents a TLS 1.3 connection.
 
-If you can't avoid hardcoding a protocol version, we strongly recommend that you specify TLS 1.2. For guidance on identifying and removing TLS 1.0 dependencies, download the [Solving the TLS 1.0 Problem](https://www.microsoft.com/download/details.aspx?id=55266) white paper.
+If you can't avoid hardcoding a protocol version, we strongly recommend that you specify TLS 1.3. For guidance on identifying and removing TLS 1.0 dependencies, download the [Solving the TLS 1.0 Problem](https://www.microsoft.com/download/details.aspx?id=55266) white paper.
```

Comment:
doing only 13 seems problematic as that can lead to compatibility issues. Especially for legacy apps. Maybe phrase it as "currently considered secure" - `Tls12 | Tls13` as of 2024.